Xamarin.MacDev/MobileProvisionIndex.cs

```diff
@@ -193,7 +193,23 @@ public int Compare (ProvisioningProfile x, ProvisioningProfile y)
 		public DateTime LastModified { get; private set; }
 		public int Version { get; private set; }
 
-		MobileProvisionIndex ()
+        static MobileProvisionIndex()
+        {
+            string xamarinFolder;
+
+            if (Environment.OSVersion.Platform == PlatformID.Unix || Environment.OSVersion.Platform == PlatformID.MacOSX)
```

Comment:
This does not follow our coding guidelines (opening brace on the same line as the if line for instance).